package.json

```diff
@@ -11,7 +11,6 @@
     "webmaker"
   ],
   "dependencies": {
-    "langmap": "0.0.14",
```

Comment:
this is our npm library, not ali's, so the real fix here is to simply update langmap and push a new version ourselves, not wait for @alicoding to take time out of his own job to help us fix a problem we can already fix =)